src/shared/configureStore.js

```diff
@@ -16,10 +16,12 @@ export default function configureStore(initialState = window.STATE_FROM_SERVER)
     middlewares.push(createLogger());
   }
 
+  const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
```

Comment:
Not sure what you're trying to do here. If you want to add a dev tool, it's better to define it as an additional enhancer and then add it to the compose function.
Example (there are more things, like middleware, but it gives an idea about the order to add things to the store):
```
const additionalStoreEnhancers = [
    typeof window === 'object' && typeof window.devToolsExtension !== 'undefined' ? window.devToolsExtension() : f => f
  ];

  const store = createStore(
    reducer,
    initialState,
    compose(autoRehydrate(), applyMiddleware(thunkMiddleware, createLogger()), ...additionalStoreEnhancers)
  );
```